### PR TITLE
Add Jetson TX2 config option 6

### PR DESCRIPTION
**Pull request by AeroNotix**
Initial work on creating a Jetson TX2 board configuration with "Configuration Option 6" from the Nvidia developer's manual.

### CODE REVIEW

File: layers/meta-balena-jetson/conf/machine/jetson-tx2-6.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#@TYPE: Machine
+##@NAME: chk5-tx2
```
Comment (joshughes) [inline layers/meta-balena-jetson/conf/machine/jetson-tx2-6.conf]: Name

**Comment by floion** [comment]
Hi, thank you for the contribution. Firstly, please make sure you follow our contribution guideline as can be seen in https://github.com/balena-os/balena-jetson-skx2#contributing

Secondly, you can use older PRs from this repo as a reference here. This is a good reference. Also, you can include a commit for your custom ODMDATA if you need it, and this can also be seen in an older commit such as this one: https://github.com/balena-os/balena-jetson-skx2/pull/146

Also, do not forget to include an icon for your board, like this commit: https://github.com/balena-os/balena-jetson-skx2/pull/140

**Comment by floion** [comment]
@resin-jenkins ok to test

**Comment by floion** [comment]
Hi, this will be handled by means of the supervisor through the dashboard. See https://github.com/balena-io/balena-supervisor/issues/1206

**Pull request closed without merging**